src/legacy/core_plugins/vis_default_editor/public/components/controls/rows_or_columns.tsx

```diff
@@ -28,8 +28,11 @@ const PARAMS = {
   COLUMNS: 'visEditorSplitBy__false',
 };
 
-function RowsOrColumnsControl({ agg, setAggParamValue }: AggControlProps) {
-  const idSelected = `visEditorSplitBy__${agg.params.row}`;
+function RowsOrColumnsControl({ editorStateParams, setStateParamValue }: AggControlProps) {
+  if (editorStateParams.row === undefined) {
+    editorStateParams.row = true;
```

Comment:
We do not prefer using object mutations in React,
it may lead to issues which are hard to find then.